src/modules/Host.rb

```diff
@@ -362,8 +358,7 @@ def load_hosts
     @hosts.load
 
     # save hosts to check for changes later
-    @hosts_init = CFA::Hosts.new
-    @hosts_init.load
+    @hosts_init = @hosts.clone
```

Comment:
Ok, if you decide to keep this, what about...

* ... to choose a more meaningful name? `previous_hosts_entries`? `initial_hosts`?
* ... to finally delete the `@modified` occurrences, in order to have a single point to check if `hosts` were modified?

Reply:
NP: BTW, this change probably is out of the scope of this commit (Refactored Host::Summary), isn't it?

Reply:
it is needed. it is used for checking modifications (and only there). If other changes didn't speed things enough I'd move it into ```Modified``` method, but it would slow down this method and it would be only moving the problem elsewhere.

Reply:
refactoring ```Host::Summary``` is not out of scope ... it is used when using yast CLI